src/Bicep.Core/Semantics/Namespaces/SystemNamespaceSymbol.cs

```diff
@@ -389,9 +404,113 @@ public class SystemNamespaceSymbol : NamespaceSymbol
                 .WithReturnType(LanguageConstants.String)
                 .WithDescription("Returns a value in the format of a globally unique identifier. **This function can only be used in the default value for a parameter**.")
                 .WithFlags(FunctionFlags.ParamDefaultsOnly)
+                .Build(),
+
+            new FunctionOverloadBuilder("loadTextContent")
+                .WithDescription("Load content of specified file into a string. Content loading occurs during compilation, not in runtime. Maximum allowed content size is 131,072 characters (including line endings).")
+                .WithRequiredParameter("filePath", LanguageConstants.String, "File's path which content will be loaded")
+                .WithOptionalParameter("encoding", LanguageConstants.String, "File encoding. If not provided, Bicep will try to automatically detect the encoding of a file based on the presence of byte order marks.")
```

Comment:
>LanguageConstants.String [](http://example.com/codeflow?start=51&length=24)

The supported encodings should be an enum (union type of literal strings in the Bicep type system)

Reply:
done.